Makefile

```diff
@@ -0,0 +1,40 @@
+PROJ_NAME=Velero
+JEKYLL_VERSION=3.5
+IMAGE_NAME=doc-test
+
+DIR := ${CURDIR}
+
+IMAGE_TAG ?= latest
+IMAGE = gcr.io/heptio-images/$(IMAGE_NAME)
+GIT_REF ?= $(shell git rev-parse --short=8 --verify HEAD)
+
+DOCKER ?= docker
+
+image:
+	$(DOCKER) build -t $(IMAGE):$(GIT_REF) -t $(IMAGE):$(IMAGE_TAG) .
+
+push: image
+	$(DOCKER) push $(IMAGE):$(IMAGE_TAG)
+	$(DOCKER) push $(IMAGE):$(GIT_REF)
+
+test:
```

Comment:
1 more change here (that would help in the upstream Makefile, too) - this like can be `test: image`, which means running `make test` will run `make image` automatically for the user, so they don't have to type both commands. Not huge, though.

Reply:
I'll make this change.